src/main/java/org/apache/commons/codec/binary/Base32.java

```diff
@@ -124,11 +124,6 @@ public class Base32 extends BaseNCodec {
     // The private member fields below are used with the new streaming approach, which requires
     // some state be preserved between calls of encode() and decode().
 
-    /**
-     * Place holder for the bytes we're dealing with for our based logic.
-     * Bitwise operations store and extract the encoding or decoding from this variable.
-     */
```

Comment:
Note to self: check out in git history if there was a variable below this comment before, or if this comment was intended for a block of variables?

Reply:
This comment was for `bitWorkArea` — https://github.com/apache/commons-codec/commit/d6f305d2efd7560634217a25f3bbee59299844dd

That field was removed in CODEC-96, so safe to remove this comment now :detective:  — https://github.com/apache/commons-codec/commit/3f41cf1609fae7c8848beb0655bcf67132c7a277